src/components/TraceDetail.tsx

```diff
@@ -253,6 +255,37 @@ function TraceDetail({
     estimateSize: () => 28,
   });
 
+  // Resize handlers for the column divider
+  const onMouseDownDivider = React.useCallback((e: React.MouseEvent<HTMLDivElement>) => {
+    isResizingRef.current = true;
+    e.preventDefault();
+  }, []);
+
+  React.useEffect(() => {
+    const controller = new AbortController();
+    const onMouseMove = (e: MouseEvent) => {
+      if (!isResizingRef.current) {
+        return;
+      }
+      const container = parentRef.current as unknown as HTMLElement | null;
+      if (!container) {
+        return;
+      }
+      const bounds = container.getBoundingClientRect();
+      const relativeX = e.clientX - bounds.left;
+      const percent = Math.min(80, Math.max(15, (relativeX / bounds.width) * 100));
+      setLeftColumnPercent(percent);
+    };
+    const onMouseUp = () => {
+      if (isResizingRef.current) {
+        isResizingRef.current = false;
+      }
+    };
+    window.addEventListener('mousemove', onMouseMove, { signal: controller.signal });
```

Comment:
Because of the signal you can of course inline the functions.
Sorry, should have mentioned that was intent.